integration_test/github_full.sh

```
@@ -0,0 +1,162 @@
+  # otherwise we assume the file did not flow through `oaspec
+  # generate`.
+  while IFS= read -r f; do
+    if ! head -n1 "$f" | grep -q "// Code generated by oaspec"; then
+      fail "Generated file $f is missing the codegen provenance header."
+    fi
+  done <<< "$files"
+
+  # `oaspec generate` runs `gleam format` during emission, so a
+  # no-op `gleam format --check` must succeed. A formatter regression
+  # surfaces here even when the build still passes.
+  # shellcheck disable=SC2086 -- intentional word-splitting on find output
```

Shellcheck (0.11.0)

[error] Couldn't parse this shellcheck directive. Fix to allow more checks.

(SC1073)